addons/account/static/src/js/account_followup_report_widgets.js

```diff
@@ -0,0 +1,165 @@
+        $('#nextActionModal').on('hidden.bs.modal', function (e) {
+            $(this).find('form')[0].reset();
+        });
+        $('#nextActionModal').modal('show');
+    },
+    onChangeBlocked: function(e) {
+        e.stopPropagation();
+        e.preventDefault();
+        var checkbox = $(e.target).is(":checked")
+        var target_id = $(e.target).parents('tr').data('id');
+        var model = new openerp.Model('account.move.line');
+        model.call('write', [[parseInt(target_id)], {'blocked': checkbox}])
+    },
+    onKeyPress: function(e) {
+        var report_name = $("div.page").attr("data-report-name");
+        if ((e.which === 13 || e.which === 10) && (e.ctrlKey || e.metaKey) && report_name == 'followup_report') {
+            $('a.btn-primary.followup-email').trigger('click');
+            var letter_context_list = [];
+            $('a.btn-primary.followup-letter').each(function() {
+                letter_context_list.push($(this).attr('context'))
+            });
+            window.open('?pdf&letter_context_list=' + letter_context_list, '_blank');
+            window.open('?partner_done=all', '_self');
+        }
+    },
+    skipPartner: function(e) {
+        var partner_id = $(e.target).attr("partner");
+        var model = new openerp.Model('res.partner');
+        if ($(e.target).is('a.btn-primary.followup-skip')) {
+            return model.call('update_next_action', [[parseInt(partner_id)]]).then(function (result) {
+                window.open('?partner_done=' + partner_id, '_self');
+            });
+        }
+        else {
+            window.open('?partner_skipped=' + partner_id, '_self');
+        }
+    },
+    printFollowupLetter: function(e) {
+        e.stopPropagation();
+        e.preventDefault();
+        var url = $(e.target).attr("target");
+        window.open(url, '_blank');
+        if ($(e.target).is('.btn-primary')) {
+            var $skipButton = $(e.target).siblings('a.followup-skip');
+            var buttonClass = $skipButton.attr('class');
+            buttonClass = buttonClass.replace('btn-default', 'btn-primary');
+            $skipButton.attr('class', buttonClass);
+            $skipButton.text('Done');
+            buttonClass = $(e.target).attr('class');
+            buttonClass = buttonClass.replace('btn-primary', 'btn-default');
+            $(e.target).attr('class', buttonClass);
+        }
+    },
+    sendFollowupEmail: function(e) {
+        e.stopPropagation();
+        e.preventDefault();
+        var context_id = $(e.target).parents("div.page").attr("data-context");
+        var contextModel = new openerp.Model('account.report.context.followup');
+        return contextModel.call('send_email', [[parseInt(context_id)]]).then (function (result) {
+            if (result == true) {
+                window.$("div.page:first").prepend(openerp.qweb.render("emailSent"));
+                if ($(e.target).is('.btn-primary')) {
```

Comment:
I guess it's the same here, only need 2 lines to change a class to another
$(e.target).toggleClass('btn-primary btn-default')
$(.followup-skip').toggleClass('btn-primary btn-default')